specifyweb/frontend/js_src/lib/userinfo.js

```diff
@@ -7,7 +7,7 @@ const $ = require('jquery');
 
     var userInfo = {};
     initialContext.load('user.json', function(data) {
-        if(typeof data.user === 'undefined'){
```

Comment:
The attribute is data.agent not data.user. Also it will be null, not undefined.

Reply:
Oh, I tested the case when the error was supposed to pop up, but did not test the opposite case 😃